public/swagger.v1.json

```diff
@@ -232,6 +232,14 @@
             "description": "Limit of result\n\nmaximum: setting.ExplorePagingNum",
             "name": "limit",
             "in": "query"
+          },
```

Comment:
Never touched this file manually. It was generated by `make generate-swagger` (exactly `mingw32-make generate-swagger`).